### DISCUSSION TITLE
Make it possible to "install" Phoenix on Chrome OS

### DISCUSSION THREAD

**Discussion by Crodattilo** [phcode-dev/phoenix#693] [Ideas]
This web app is already amazing.
It would be cool to code websites on chromebooks with the option to install Phoenix through the "install" feature on Chrome browser.
For sure I can create a simple shortcut but it would be quicker and easier : )

**Comment by abose** [comment]
This Could be a good feature to have just for Chrome OS.

The reason it is not currently provided is that we are currently working on native builds for Win, Mac and, Linux.

Would prefer if there is a native option for chrome OS. If that is not available, Will enable the "install" feature in chromeOS.

Tacking Issue: https://github.com/phcode-dev/phoenix/issues/694
